src/interfaces/vault.spec.ts

```diff
@@ -694,6 +702,114 @@ describe("VaultInterface", () => {
     });
   });
 
+  describe("getDepositAllowance", () => {
+    it("should fetch token deposit allowance", async () => {
+      const account: Address = "0xAccount";
+      const vault: Address = "0xVault";
+      const token: Address = "0xToken";
+      const spender: Address = "0xSpender";
```

Comment:
Let's move these higher up since whey are just `const` and used in multiple tests

Reply:
Fixed in https://github.com/yearn/yearn-sdk/pull/266/commits/8a1b0ccc53c2a9aba0f3ffc21142320107cf2c2d